js/exercise_one/index.js

```diff
@@ -0,0 +1,144 @@
+function sortTableByCost(table) {
+  const tBody = table.tBodies[0];
+  const rows = Array.from(tBody.querySelectorAll("tr"));
```

Comment:
Try to be consistent with the usage of single and double quotes. We usually use single for everything